src/GovUk.Education.ExploreEducationStatistics.sln

```diff
@@ -94,6 +94,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GovUk.Education.ExploreEduc
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Requests", "GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Requests\GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Requests.csproj", "{A29A7A00-47E8-4D58-B461-505B88AE66BC}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Public", "Public", "{1A815444-657A-4234-9871-3D5936502AB0}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Admin", "Admin", "{7B1A4E2D-FCE7-4079-8005-F081C48C67D7}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AdminAPI", "AdminAPI", "{F9721FCB-09B7-4A2D-A69F-D32F5CC2D43C}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Shared", "Shared", "{A58A13A6-EA97-4CE1-A134-83045CC08C85}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ContentAPI", "ContentAPI", "{5E6BE0C0-7B4F-43A6-85A7-91C146E96049}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Processor", "Processor", "{927B83FD-01E9-4983-9EBB-975125E66595}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Global", "Global", "{20631758-CF58-4E2D-BEB1-D9F18E5D2F10}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Notifier", "Notifier", "{BC0265FB-2B8F-4964-A14B-401246CF795B}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "DataAPI", "DataAPI", "{C9AC27EE-2D41-4285-91BA-4126BDC42E33}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Processor", "Processor", "{5828FACF-4A73-4D56-B36B-80843BAC4CA1}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Publisher", "Publisher", "{BBA913B5-8D13-462C-AFE9-162B31F769BF}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "DataServices", "DataServices", "{7F6E631B-A1E2-45E6-BE32-B020B58C0E7D}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Utils", "Utils", "{5FD84034-5CD0-4F43-9B6D-6BEFF66864FF}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "PublicAPI", "PublicAPI", "{A463EAD4-4E6F-4E26-B519-2EA645E60E66}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "API", "API", "{0F77425C-2F87-481E-94EB-4FA2B47EF53D}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ContentServices", "ContentServices", "{B980CD0D-48D4-485F-933B-0681D81E408E}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "PublicDataProcessor", "PublicDataProcessor", "{82AD39EC-E1A5-44A0-9280-EB6F7ABB0112}"
```

Comment:
I started off by thinking some top level groups for each service or component such as

```
Admin
Content
Data
Notifier
Processor
Publisher
PublicApi
Shared
```

to group the projects might work better and avoid having to pigeonhole them into one of Admin, Public, Shared, or Global, but I think this approach is fine!